docs/widgets.md

```diff
@@ -358,6 +358,55 @@ component Example inherits Window {
 }
 ```
 
+## `StandardTableView`
+
+The `StandardTableView` represents a table of data with columns and rows. As cell model items it uses the `StandardListViewItem`.
+
+### Properties
+
+Same as ListView, and in addition:
+
+* **`current-sort-column`** (*int*): Indicates the sorted column. -1 mean no column is sorted.
+* **`columns`** (*`[TableColumn]`*): Defines the model of the table columns.
+* **`rows`** (*`[StandardListViewItem]`*): Defines the model of table rows.
```

Comment:
TableColumn and StandardListViewItem are not documented.
(StandardListViewItem is mentioned in the docs for the StandardListView, but otherwise not documented)